ztp/ztp-ran-policy-generator/README.md

```diff
@@ -1,11 +1,265 @@
+
+```
+apiVersion: policy.open-cluster-management.io/v1
+kind: Policy
+metadata:
+    name: group-du-ptp-config-policy
+    namespace: groups-ran-sub
+    annotations:
+        policy.open-cluster-management.io/categories: CM Configuration Management
+        policy.open-cluster-management.io/controls: CM-2 Baseline Configuration
+        policy.open-cluster-management.io/standards: NIST SP 800-53
+spec:
+    remediationAction: enforce
+    disabled: false
+    policy-templates:
+        - objectDefinition:
+            apiVersion: policy.open-cluster-management.io/v1
+            kind: ConfigurationPolicy
+            metadata:
+                name: group-du-ptp-config-policy-config
+            spec:
+                remediationAction: enforce
+                severity: low
+                namespaceselector:
+                    exclude:
+                        - kube-*
+                    include:
+                        - '*'
+                object-templates:
+                    - complianceType: musthave
+                      objectDefinition:
+                        apiVersion: ptp.openshift.io/v1
+                        kind: PtpConfig
+                        metadata:
+                            name: slave
+                            namespace: openshift-ptp
+                        spec:
+                            recommend:
+                                - match:
+                                    - nodeLabel: node-role.kubernetes.io/worker-du
+                                  priority: 4
+                                  profile: slave
+                            profile:
+                                - interface: ens5f0
+                                  name: slave
+                                  phc2sysOpts: -a -r -n 24
+                                  ptp4lOpts: -2 -s --summary_interval -4
+                                  ptp4lConf: |
+                                    [global]
+                                    #
+                                    # Default Data Set
+                                    #
+                                    twoStepFlag 1
+                                    slaveOnly 0
+                                    priority1 128
+                                    .....
+```
+
+# Install and execute
 -  We assume kustomize and golang are installed
```

Comment:
Do we need to list the ACM dependency here?

Reply:
no need for ACM dependency, just kustomize and golang